great_expectations/dataset/pandas_dataset.py

```diff
@@ -3,7 +3,7 @@
 import inspect
 import json
 import re
-from datetime import datetime
+from datetime import datetime # , timedelta # Add for case of testing timedelta types
```

Comment:
?

Reply:
Added as a note about the placeholder for potentially adding additional types accepted by this expectation, but thought of it as out of scope for the current PR (which is optimization focused)